context/app/static/js/helpers/init-trackers.js

```diff
@@ -1,41 +1,37 @@
-import { init as sentryInit } from '@sentry/react';
 import { TracingInstrumentation } from '@grafana/faro-web-tracing';
 import { LogLevel, getWebInstrumentations, initializeFaro } from '@grafana/faro-web-sdk';
+import { ReactIntegration } from '@grafana/faro-react';
 
 /**
- * Initialize Faro and Sentry tracking
+ * Initialize Faro trackers
  * This function should be used in the main `index.jsx` before the app is rendered.
  */
 function initTrackers() {
-  const release = `portal-ui-react@${PACKAGE_VERSION}`;
-  const enableTracking = ['prod', 'prod-stage'].includes(sentryEnv);
-
-  sentryInit({
-    dsn: sentryDsn,
-    environment: sentryEnv,
-    enabled: enableTracking,
-    release,
-  });
+  const version = `portal-ui-react@${PACKAGE_VERSION}`;
+  const environment = sentryEnv;
+  const paused = !['prod', 'prod-stage'].includes(environment);
```

Comment:
Can we distinguish what's coming from prod vs stage? If not, perhaps we should only show prod so it's clear what's being monitored and reported on.

Reply:
There is an `app_environment` value reported when events are pushed in grafana:
![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/1ec27534-00ca-4767-abd2-dba6c670e9c0)

It seems like it's not being correctly populated from our environment variable, however, since all my events are being reported as `local` while having the relevant environment variable set to `prod`, so I'll experiment with the configuration more.

Reply:
I confirmed that the problem wasn't with the value being passed to `app_environment`, but with the `paused` prop not stopping reporting like I expected it to. I've revised the approach to suppressing events to use the `beforeSend` middleware instead, which checks if the current environment is prod or prod-stage and only reports to faro in those envs.